CONTRIBUTING.md

```diff
@@ -1,123 +1,126 @@
-Thank you for considering contributing to the NEAR reference client!
+Thank you for your interest in contributing to the NEAR reference client!  We
+welcome contributions from everyone.  Below are various bits of information to
+help you get started.  If you require additional help, please reach out to us on
+our [zulip channel](https://near.zulipchat.com/).
 
-We welcome all external contributions. This document outlines the process of contributing to nearcore.
-For contributing to other repositories, see `CONTRIBUTING.md` in the corresponding repository.
-For non-technical contributions, such as e.g. content or events, see [this document](https://docs.nearprotocol.com/docs/contribution/contribution-overview).
+# Getting started
 
-For an overview of the NEAR core architecture, see [this playlist](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
+- For non-technical contributions, such as e.g. content or events, see [this
+document](https://docs.nearprotocol.com/docs/contribution/contribution-overview).
 
-# Pull Requests and Issues
+- For an overview of the NEAR core architecture, see [this
+playlist](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
 
-All the contributions to `nearcore` happen via Pull Requests. To create a Pull Request, fork `nearcore`, create a new branch, do the work there, and then send the PR via the Github interface.
+- If you have never contributed to nearcore before and are looking for
+relatively simple issues to get started with, please checkout issues labeled
+with `C-good-first-issue`
+[here](https://github.com/near/nearcore/labels/C-good-first-issue).  If you see
+one that looks interesting and is not claimed, please comment on the issue that
```

Comment:
I would consider removing ‘and is not claimed’.  We may have issues which are claimed but the author haven’t touched it for months and presumably abandoned it.